src/shaderlib/project/project.glsl.js

```diff
@@ -46,6 +48,9 @@ float project_scale(float meters) {
 }
 
 vec2 project_scale(vec2 meters) {
+  if (projectionMode == PROJECT_UTM_OFFSETS) {
```

Comment:
Though it's obvious right now, let's add a comment here stating that in UTM_OFFSETS mode, lat and lon are no longer independent in the scaling transformation